app/services/webhook_service.py

```diff
@@ -457,12 +474,28 @@ async def process_contents_fetch_event(webhook_request_data: WebhookRequestData)
     async_contents: Dict = await fetch_async_contents_by_id(id=contents_fetch_event_job_id)
 
     if not async_contents:
-        logging.error(f"Profile-analytics do not exists with publish-id: {contents_fetch_event_job_id}")
+        logging.error(f"Contents do not exists with publish-id: {contents_fetch_event_job_id}")
         return
 
     await send_events(webhook_event=webhook_request_data.event, data=async_contents)
 
 
+async def process_professional_contents_fetch_event(webhook_request_data: WebhookRequestData):
+    job_id = webhook_request_data.data.get("job_id")
+
+    professional_contents_fetch_event = ProfessionalContentsFetchEvent(job_id=job_id)
+
+    professional_contents_fetch_event_job_id = professional_contents_fetch_event.job_id
+
+    professional_contents: Dict = await fetch_async_professional_contents_by_id(id=professional_contents_fetch_event_job_id)
+
+    if not professional_contents:
+        logging.error(f"Professional-Contents do not exists with publish-id: {professional_contents_fetch_event_job_id}")
+        return
```

Comment:
just simplifying things, nothing major
```suggestion
    event = ProfessionalContentsFetchEvent(**webhook_request_data.data)

    professional_contents: Dict = await fetch_async_professional_contents_by_id(id=event.job_id)

    if not professional_contents:
        logging.error(f"Professional-Contents do not exists with publish-id: {event.job_id}")
        return
```